Modules/Forum/classes/class.ilObjForumListGUI.php

```diff
@@ -60,133 +52,135 @@ public function init()
         $this->commands = ilObjForumAccess::_getCommands();
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function getProperties()
+    public function getProperties() : array
     {
-        if (!$this->access->checkAccess('read', '', $this->ref_id)) {
-            return array();
+        global $DIC;
```

Comment:
Could you please try to move the `global $DIC` to `__construct()`, use member variables for the required services, and use this member variables here.